.lintstagedrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "*.{js,jsx}": ["eslint --fix", "git add"],
+  "*.{json,md}": ["prettier --write", "git add"]
+}
```

Comment:
I tried changing a js file so a code line goes over 80 chars and Prettier did reformat it, however the commit didn't fail. We need to figure out a way to stop the commit if any tool makes changes to the files ([`--bail` behavior](https://github.com/azz/pretty-quick#--bail) of `pretty-quick`). (Would also be good to keep [`--verbose`](https://github.com/azz/pretty-quick#--verbose) too).

Reply:
Wait. This code is outdated, let me comment again with the most current code....